types/rapid-hotels-api/CompleteHotelInformation.ts

```diff
@@ -0,0 +1,7 @@
+import { APIHotelDetailsJSONFormatted } from "@/app/api/hotels/details/route";
+import { APIHotelRoomOffersJSONFormatted } from "@/app/api/hotels/search/rooms/route";
+
+export type CompleteHotelInfo = {
+    hotelDetails: APIHotelDetailsJSONFormatted | null;
+    hotelRooms: APIHotelRoomOffersJSONFormatted | null;
+} | null;
```

Comment:
Are we sure, they may be null. If not, it's best to leave that out. Otherwise, we'll need to add if statements to ensure the values are present.